src/frontend/src/lib/components/ui/ContentWithButtons.svelte

```diff
@@ -0,0 +1,3 @@
+<slot class="stretch" />
+
+<slot name="buttons" />
```

Comment:
What do you think about naming this slot `toolbar` or something similar? If yes, don't know if the cmp should reflects it?

Just an idea, already approving the PR.

Reply:
So, if it's `toolbar` the component will be `ContentWithToolBar`, correct?

Reply:
Ah why not, sounds cool no?

Reply:
Yes, better! Eheh! Changed